site/docs/concepts/web-app.md

```diff
@@ -174,15 +183,20 @@ You use the table to monitor your materializations. It's nearly identical to the
 
 **2:** Enable, Disable, and Delete buttons. These actions will be applied to the selected table rows. Choose **Disable** to temporarily pause the flow of data, **Enable** to resume, and **Delete** to permanently remove the materialization(s).
 
-**3:** Filter materializations by name.
+**3:** Filter materializations by name. Type a catalog prefix or unique materialization name to return materializations that match your query.
```

Comment:
>Filter captures. Type a catalog prefix, unique capture name, or connector name to return captures that match your query.

The same filter criteria applies for the materialization table. You can filter by connector name here as well.